tools/ci/docker/engine/Dockerfile

```
@@ -69,6 +70,8 @@ ENV LOG_LEVEL=${LOG_LEVEL}
 ENV PRINT_VERSION=${PRINT_VERSION}
 ENV JAVA_TOOL_OPTIONS=${JAVA_TOOL_OPTIONS}
 ENV ENSO_LAUNCHER=${ENSO_LAUNCHER}
+ENV LANGUAGE_SERVER_YDOC_HOSTNAME=${YDOC_HOST}
+ENV LANGUAGE_SERVER_YDOC_PORT=${YDOC_PORT}
```

Why this line isn't just 
```suggestion
ENV LANGUAGE_SERVER_YDOC_PORT="0.0.0.0"
```

---

That wouldn't allow you to override the port. Also, `0.0.0.0` is not a port.

---

Off by one error. I hope you understand the intention.